client/authority-discovery/src/worker.rs

```diff
@@ -154,30 +151,31 @@ where
 		dht_event_rx: DhtEventStream,
 		role: Role,
 		prometheus_registry: Option<prometheus_endpoint::Registry>,
+		config: crate::WorkerConfig,
 	) -> Self {
-		// Kademlia's default time-to-live for Dht records is 36h, republishing records every 24h.
+		// Kademlia's default time-to-live for Dht records is 36h, republishing records every 12h by default.
```

Comment:
```suggestion
		// Kademlia's default time-to-live for Dht records is 36h, republishing records every 24h through libp2p-kad.
```

`libp2p-kad` republishes values every 24h. In addition, the authority discovery module re-triggers publishing every 12h to make sure values persist across restarts.

See [libp2p-kad/behaviour](https://github.com/libp2p/rust-libp2p/blob/421ae9d9c9cfe1398f8bf6e2dd7d90c8e8f36018/protocols/kad/src/behaviour.rs#L257-L269) for details.